Gems/ROS2/Code/Source/RobotImporter/URDF/UrdfParser.h

```diff
@@ -75,8 +76,8 @@ namespace ROS2
             //! Stores the modified URDF content after parsing, empty if no modification occurred
             std::string m_modifiedURDFContent;
 
-            //! Stores the modified URDF tags after parsing, empty if no modification occurred
-            AZStd::vector<AZStd::string> m_modifiedURDFTags;
+            //! Stores describtion of URDF modifications, empty if no modification occurred
+            Utils::UrdfModifications m_UrdfModifications;
```

Comment:
The same change applies for the rest of usages
```suggestion
            Utils::UrdfModifications m_urdfModifications;
```